.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,36 @@
+name: Documentation bug
+description: Raise an issue related to content.
+title: "📝 Documentation Bug or documentation feature: "
```

Comment:
Close!
What I meant earlier was, if we add an emoji here, shouldn't add emojis on current bug/feature templates?
Also something shorter like `📝 Documentation:` or `📝 Doc:` so we can keep the title? (and some consistency with our current templates)